presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Add feature and session toggles for *top-N row number* optimization.
+* Add ``ngrams`` function to generate the N-grams from an array.
+* Add ``qdigest`` type and associated functions.
+* Add statistics calculator support for ``TINYINT``, ``SMALLINT`` and ``VARBINARY`` literals.
+* Add system to delay query execution until a minimum number of workers nodes are available. If the requirement is not met within a time limit, the query is failed.
+* Improve error message for broken :func:``array_sort`` comparator.
+* Fix memory leak related to query tracking in coordinator. Completed queries are supposed to be purged shortly after its completion. However, a 0.213 regression broke this.
+* Lazily build hash table for ``MapBlock`` when the hash table is used in ``SingleMapBlock``.
+* Make planning more stable with regard to query predicates and joins.
+* Make distributed ``ORDER BY`` and aggregation unspill progress on early yield signals.
+* Make *top-N row number* optimization work in some cases when columns are pruned.
+* Print distribution type for semi-joins in explain plans.
```

Comment:
Print -> Include

Reply:
Also, it should be “in EXPLAIN output”. There’s no such a thing as “explain plans”